zips/zip-0233.md

```diff
@@ -19,224 +20,103 @@ The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED", "OPTIONAL",
-First, the `ZSF_DEPOSIT[H]` is calculated based solely on `ZSF_BALANCE[H-1]`.
-This is subtracted from the previous block's balance to be distributed as part
-of the block reward. Second, the sum of all the `ZSF_DEPOSIT` fields of all
-transactions in the block is added to the balance.
+The burned value must now be considered when calculating the total output value
+of a transaction. It should be treated similarly to a transparent output,
+except that there is no way for this value to be used as an input in a future
+transaction.
 
+## Transaction Format Requirements
 
-### Non-Coinbase Transactions
-
-If the `ZSF_DEPOSIT` field is not present in an older transaction version, it
-is defined to be zero for non-coinbase transactions.
-
-#### Consensus Rule Changes
-
-- Transparent inputs to a transaction insert value into a transparent
-  transaction value pool associated with the transaction. Transparent outputs
-  **and sustainability fund deposits** remove value from this pool.
-
-### Coinbase Transactions
-
-Any excess miner fee/reward is sent to the ZSF.
-
-In new blocks, this deposit is tracked via the `ZSF_DEPOSIT` field in coinbase
-transactions. 
-
-If the `ZSF_DEPOSIT` field is not present in a coinbase transaction with an
-older transaction version, it is defined as the value of any unspent miner fee
-and miner reward. This re-defines these previously unspendable funds as ZSF
-deposits.
-
-#### Consensus Rule Changes
-
--  The remaining value in the transparent transaction value pool of a coinbase
-   transaction is **deposited in the sustainability fund**.
-
-### `ZSF_DEPOSIT` Requirements
-
-- ZIP 230 [3] must be updated to include `ZSF_DEPOSIT`.
-- ZIP 244 [4] must be updated as well to include `ZSF_DEPOSIT`.
+* ZIP 230 [3] must be updated to include the new `burnAmount` field.
+* ZIP 244 [4] must be updated as well to include the new `burnAmount` field.
 
 # Rationale
 
 All technical decisions in this ZIP are balanced between the necessary
-robustness of the ZSF mechanics, and simplicity of implementation. 
-
-## `ZSF_BALANCE` as node state
-
-Tracking the `ZSF_BALANCE` value as a node state using the above formula is
-very simple in terms of implementation, and should work correctly given that
-the node implementations calculate the value according to the specifications.
+robustness of the NSM mechanics, and simplicity of implementation.
 
-## `ZSF_DEPOSIT` as explicit transaction field
+## New transaction field for burn amount
 
-An explicit value distinguishes the ZEC destined to Sustainability Fund
-deposits from the implicit transaction fee. Explicitness also ensures any
-arithmetic flaws in any implementations are more likely to be observed and
-caught immediately.
+An explicit value distinguishes the burned ZEC from the implicit transaction
+fee. Explicitness also ensures any arithmetic flaws in any implementations are
+more likely to be observed and caught immediately.
```

Comment:
It is unlikely that the transaction fee will remain implicit in the new transaction version. The current implicitness of the fee is also not relevant to the point being made; it's sufficient to say that the burned value needs to be distinguished from the fee, whether the latter is implicit or explicit.
```suggestion
An explicit value distinguishes the burned ZEC from the transaction fee.
Explicitness also ensures any arithmetic flaws in implementations are more
likely to be observed and caught immediately.
```

Reply:
Applied in [this commit](https://github.com/zcash/zips/pull/913/commits/fad4eae55eecb19a19dcac999c2b2a214887b596) :heavy_check_mark: